modules/extension/wmts/src/main/java/org/geotools/ows/wmts/map/WMTSCoverageReader.java

```diff
@@ -346,14 +338,23 @@ GridCoverage2D getMap(ReferencedEnvelope requestedEnvelope, int width, int heigh
      * @throws TransformException
      */
     private ReferencedEnvelope toEastNorthAxisOrder(ReferencedEnvelope envelope)
-            throws FactoryException, TransformException {
-        CoordinateReferenceSystem crsExtent = envelope.getCoordinateReferenceSystem();
-        if (CRS.getAxisOrder(crsExtent) == CRS.AxisOrder.NORTH_EAST) {
-            String srsExtent = CRS.toSRS(crsExtent);
-            crsExtent = CRS.decode(srsExtent, true);
-            envelope = envelope.transform(crsExtent, false);
+            throws FactoryException {
```

Comment:
In the javadoc above, while you're at it, "eith" should be "with"

Reply:
fixed